lib/shimmer.js

```diff
@@ -555,18 +572,13 @@ function instrumentPostLoad(agent, nodule, moduleName, resolvedName, esmResolver
     })
 
     applyDebugState(shim, resolvedNodule, esmResolver)
-    trackInstrumentationUsage(
-      agent,
-      shim,
-      instrumentation.specifier || moduleName,
-      NAMES.FEATURES.INSTRUMENTATION.ON_REQUIRE
-    )
+    trackInstrumentationUsage(agent, shim, moduleName, NAMES.FEATURES.INSTRUMENTATION.ON_REQUIRE)
 
     // Tracking instrumentation is only used to add the supportability metrics
     // that occur directly above this.  No reason to attempt to load instrumentation
     // as it does not exist.
     if (instrumentation.type === MODULE_TYPE.TRACKING) {
```

Comment:
is it a problem we have these types defined in two places now? shim constants and instrumentation descriptor?

Reply:
I meant to clean this up.

Reply:
I have addressed this in [`ca26505` (#2104)](https://github.com/newrelic/node-newrelic/pull/2104/commits/ca26505864a07bfb378a07d8992e3d84434313c4)